connectivity/manifests/client-egress-to-fqdns-google.yaml

```diff
@@ -30,3 +30,6 @@ spec:
     - matchLabels:
         k8s:io.kubernetes.pod.namespace: kube-system
         k8s:k8s-app: kube-dns
+    - matchLabels:
+        k8s:io.kubernetes.pod.namespace: kube-system
+        k8s:k8s-app: coredns
```

Comment:
Nit: missing new line.

Reply:
Hey @pchaigno - Was just fixing this! Should be sorted now, apologies.